setup.py

```diff
@@ -22,7 +22,7 @@
       install_requires=['numpy', 'requests'],
       python_requires='>=3.8',
       extras_require={
-        'gpu': ["pyopencl"],
+        'gpu': ["pyopencl", "six"],
```

Comment:
Why do we need this?

Reply:
I am not sure. But since 1a1c63a tests have not run due to six not being installed... I checked the PR and only an example was added, so I am not sure why it started failing. Perhaps upstream issues?

Reply:
Hmm, okay